src/app/services/xml-parsers/generic-parser.service.ts

```diff
@@ -60,6 +61,18 @@ export class GenericParserService {
     return text;
   }
 
+  public parseParagrah(xml: XMLElement): ParagraphData {
+    const paragraphComponent: ParagraphData = {
+      type: 'ParagraphComponent',
+      class: xml.tagName ? xml.tagName.toLowerCase() : '',
+      content: this.parseChildren(xml),
+      attributes: this.parseAttributes(xml),
+      n: this.parseAttributes(xml).n || '-1',
+    };
+
+    return paragraphComponent;
+  }
+
```

Comment:
What about managing a paragraph with a `<pb>` element inside?
For example:
```
<div>
   <pb xml:id="page1" />
   <p n="2">
        Lorem ipsum...
   <p>
   <p n="3">
       Ispum dolor  <pb xml:id="page2"/> sit amet sit amet sit amet....
   </p>
   <p n="4">
      [...]
   </p>
</div>
```

Reply:
`pb`s would be managed recursively by the parser

Reply:
I agree. There will be a specific parser for `<pb>`s, but is a subject for a new PR!